OrderForm/ContentViewModel.swift

```diff
@@ -8,6 +8,16 @@ class ContentViewModel: ObservableObject {
     @Published var quantity = 1
     var pizzaOption = ["Large", "Medium"] // TODO: as smiles, but it should be connected with pizzaOptionTag
     @Published var orderStatus: Bool = false
+    @Published var showingAlert = false
```

Comment:
showingAlert logic connected only with reset logic - may be better to move it to `ResetButtonSection`

Reply:
corrected